src/components/Main/Main.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+// eslint-disable-next-line max-len
```

Comment:
Consider removing the eslint-disable-next-line comment and fixing the linting issue instead. 🚨

Reply:
ok